### PR TITLE
ped: improve SET_PED_PROP_INDEX and SET_PED_COMPONENT_VARIATION docs

### PR DESCRIPTION
**Pull request by PichotM**
come correct grammar mistakes

### CODE REVIEW

File: PED/SetPedComponentVariation.md [hunk 1]
```diff
@@ -8,33 +8,31 @@ ns: PED
...
-* **drawableId**: 
-* **textureId**: 
-* **paletteId**: 
+* **ped**: The ped handle.
+* **componentId**: The component that you want to set.
+* **drawableId**: The drawable id that is going to be set.
+* **textureId**: The texture id of the drawable.
+* **paletteId**: 0 to 3.
+
+## MP Freemode list of components
```
Comment (stannum-cfx) [inline PED/SetPedComponentVariation.md]: All this (and below) must be moved above `## Parameters`. This is still a requirement for any 'description' text to be detected. Any description text MUST be placed directly below the C code block and ABOVE any other sections like `## Examples` `## Parameters` `## Return value` and similar.

Currently all this text will not end up in the description section of the docs.

File: PED/SetPedPropIndex.md [hunk 2]
```diff
@@ -5,27 +5,27 @@ ns: PED
...
-* **drawableId**: 
-* **TextureId**: 
-* **attach**: 
+* **ped**: The ped handle.
+* **componentId**: The component that you want to set.
+* **drawableId**: The drawable id that is going to be set.
+* **textureId**: The texture id of the drawable.
+* **attach**: Attached or not. 
+
+## MP Freemode list of props
```
Comment (stannum-cfx) [inline PED/SetPedPropIndex.md]: Same with this one, please move this data to the proper description section.

**Comment by PichotM** [comment]
Fixed?

**Comment by blattersturm** [comment]
For some reason I doubt second-level headings will be propagated at all. Try `###` or to be 100% certain just use bold.

**Comment by PichotM** [comment]
> For some reason I doubt second-level headings will be propagated at all. Try `###` or to be 100% certain just use bold.

Replace description `##` by `###` ?

**Comment by PichotM** [comment]
✅

**Pull request merged**